posthog-react-native/src/posthog-rn.ts

```diff
@@ -451,4 +452,29 @@ export class PostHog extends PostHogCore {
     this.setPersistedProperty(PostHogPersistedProperty.InstalledAppBuild, appBuild)
     this.setPersistedProperty(PostHogPersistedProperty.InstalledAppVersion, appVersion)
   }
+
+  /**
+   * @todo Where should this go, and should the result be cached?
+   * I can't find a public method in PostHog which does an authenticated fetch,
+   * so for now I've included the fetch here so it can access the api key.
+   */
+  public async fetchSurveys(): Promise<Survey[]> {
+    const response = await this.fetch(
+      // PostHog Dashboard complains I'm using an old SDK version, I think because the ver query param isn't provided.
+      // TODO I've pulled a recent version from posthog-js, but this should be updated.
+      `${this.host}/api/surveys/?token=${this.apiKey}&ver=1.200.0`,
+      {
+        method: 'GET',
+        headers: this.getCustomHeaders(),
+      }
+    )
+
+    if (response.status > 300) {
+      // TODO Best practice for handling this?
```

Comment:
by doing [this](https://github.com/PostHog/posthog-js-lite/pull/333/files#r1922625072), error handling is builtin